packages/wdio-config/tests/FileSystemPathService.test.ts

```diff
@@ -28,66 +33,72 @@ describe('FileSystemPathService', () => {
         })
 
         it('should return current working directory', function () {
-            var svc = new FileSystemPathService()
+            const svc = new FileSystemPathService()
             expect(svc.getcwd()).toEqual(process.cwd())
         })
 
         it('should throw if cwd returns undefined', function () {
             process.cwd = () => undefined as any as string
-            var svc = new FileSystemPathService()
+            const svc = new FileSystemPathService()
             expect(() => svc.getcwd()).toThrowError('Unable to find current working directory from process')
         })
     })
 
     describe('isFile', function () {
         it('should return true if file esxists', function () {
-            var svc = new FileSystemPathService()
+            const svc = new FileSystemPathService()
```

Comment:
```suggestion
            const svc = new FileSystemPathService()
            vi.mocked(fs.existsSync).mockReturnValue(true)
```

Reply:
yes that works, is there a way to ignore `fs.lstatSync(filepath)` from mocking